src/mame/efo/cedar_magnet.cpp

```diff
@@ -265,6 +265,440 @@ MASTER BOARD
    |                                                         __________     |
    |                                                        |MC74HC157|     |
    |________________________________________________________________________|
+*/
+
+/*
+Additional hardware notes from ArcadeHacker:
+
+
+Z80 main @ 3.9986 Mhz
+---------------------
+
+Memory map
+‘Bank bits’ are these from IC 48 PIO  (0x20)
+A5 -> 4 74HC10 3NAND IC19  // Main CPU Memory bank switch
+A6 -> 2 74HC10 3NAND IC19  // Main CPU Memory bank switch
+
+Initial state  (bank bits 11?)
+0000-1FFF eprom
+2000-3FFF eprom mirror
+4000-5FFF eprom mirror
+6000-7FFF eprom mirror
+8000-9FFF eprom mirror
+A000-BFFF eprom mirror
+C000-DFFF eprom mirror
+E000-FFFF eprom mirror
+
+Upload to other CPUs state (Bank bits 10 )
+
+This is complex and not 100% understood.
+
+ 0000 - BFFF   - RAM?  (same as in state 01 / 00)
+ C000  FFFF   - a 0x4000 sized window into the RAM for the other CPUs, uploads programs**
+
+**see 0x43 IC 49 PIO
+B2 -> 36 J7
+B3 -> 35 J7
+and MAYBE  B1 -> 3 74HC04 IC21
+2 bank bits gives a full 0x10000 (64kb view) but could be more?
+
+WHICH cpu we get a window into accessible seems to be determined by the busreq bits (main CPU must have bus?)
+IC48 A1 -> 47 J6 - output (DH: busreq plane board a?)
+IC48 B1 -> 48 J6 - output (DH: busreq plane board b?)
+IC48 B5 -> 43 J6 - output (DH: busreq sprite board?)
+IC49 B5 -> 8 J3  - output (DH: sound CPU busreq)
```

Comment:
“WHICH cpu we get a window into accessible” doesn’t make sense as written.  Please rephrase this sentence.